supervisor/shared/serial.c

```diff
@@ -69,7 +71,7 @@ bool serial_connected(void) {
 #if defined(DEBUG_UART_TX) && defined(DEBUG_UART_RX)
     return true;
 #else
-    return tud_cdc_connected();
+    return _serial_connected;
```

Comment:
Nothing.  It's here by accident.  If I rebase the branch I'll try to drop this patch.   Here's the explanation for that change from its commit message:
```
    supervisor: Improve serial connection detection
    
    These changes remove the caveat from supervisor.runtime.serial_connected.
    
    It appears that _tud_cdc_connected() only tracks explicit changes to the
    "DTR" bit, which leads to disconnects not being registered.
    
    Instead:
     * when line state is changed explicitly, track the dtr value in
       _serial_connected
     * when the USB bus is suspended, set _serial_connected to False
    
    Testing performed (using sam e54 xplained):  Run a program to show
    the state of `serial_connected` on the LED:
    ```
    import digitalio
    import supervisor
    import board
    
    led = digitalio.DigitalInOut(board.LED)
    while True:
        led.switch_to_output(not supervisor.runtime.serial_connected)
    ```
    
    Try all the following:
     * open, close serial terminal program
        - LED status tracks whether terminal is open
     * turn on/off data lines using the switchable charge-only cable
        - LED turns off when switch is in "charger" position
        - LED turns back on when switch is in Data position and terminal is
          opened (but doesn't turn back on just because switch position is
          changed)
```